content/show/linux-unplugged/0598.md

```diff
@@ -0,0 +1,157 @@
+    "🫏"
+  ],
+  "hosts": [
+    "brent",
+    "chris",
+    "wes"
+  ],
+  "guests": [],
+  "sponsors": [
+    "tailscale.com-lup",
+    "1password.com-lup"
+  ],
+  "podcast_duration": "01:23:41",
+  "podcast_file": "https://aphid.fireside.fm/d/1437767933/f31a453c-fa15-491f-8618-3f71f1d565e5/d7cb9739-9767-4628-adad-da7ce1988763.mp3",
+  "podcast_bytes": 70968307,
+  "podcast_chapters": {
+    "version": "1.1.0",
+    "chapters": [
+      {
+        "startTime": 0,
+        "title": "Intro",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 194,
+        "title": "Housekeeping",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 405,
+        "title": "The Nix Problem",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 3291,
+        "title": "Boosts",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 4453,
+        "title": "Picks",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      },
+      {
+        "startTime": 4856,
+        "title": "Outro",
+        "img": null,
+        "url": null,
+        "toc": null,
+        "endTime": null,
+        "location": null
+      }
+    ],
+    "author": "Jupiter Broadcasting",
+    "title": "598: Not Your Distrohopper's Distro",
+    "podcastName": "LINUX Unplugged",
+    "description": null,
+    "fileName": null,
+    "waypoints": null
+  },
+  "podcast_alt_file": null,
+  "podcast_ogg_file": null,
+  "video_file": null,
+  "video_hd_file": null,
+  "video_mobile_file": null,
+  "youtube_link": null,
+  "jb_url": "https://www.jupiterbroadcasting.com/show/linux-unplugged/598",
+  "fireside_url": "https://linuxunplugged.com/598",
+  "value": null
+}
+
+
+### Episode Links
+
+* [💥 Gets Sats Quick and Easy with Strike](https://strike.me/ "💥 Gets Sats Quick and Easy with Strike")
+* [📻 LINUX Unplugged on Fountain.FM](https://www.fountain.fm/show/dWiuBeqpDSM86AwXRXov "📻 LINUX Unplugged  on Fountain.FM")
+* [NixOS is interesting, but has fatal flaws](https://www.youtube.com/watch?v=x6ip1cVVr7E "NixOS is interesting, but has fatal flaws")
+* [LUP 600 meetups!](https://colonyevents.com/events/ "LUP 600 meetups!")
+* [Planet Nix](https://planetnix.com/ "Planet Nix") — March 6-7, Before and Alongside SCaLE
+* [SCALE 22x](https://www.socallinuxexpo.org/scale/22x "SCALE 22x") — March 6-9, 2025 at the Pasadena Convention Center in Pasadena, CA.
+* [LFNW2025 - "25 Years of Community Excellence"](https://linuxfestnorthwest.org/ "LFNW2025 - &quot;25 Years of Community Excellence&quot;") — April 25 - 27, 2025
```

Comment:
Do we normally have `&quot;` instead of `'` characters?

Reply:
Yes for the title attribute, otherwise it will break the rendering.

Reply:
Thank you both for being on it! :mechanical_arm:

Reply:
Cool. Just wanted to make sure